oteapi_dlite/strategies/mapping.py

```diff
@@ -41,12 +54,32 @@ class DLiteMappingStrategy:
 
     """
 
-    mapping_config: MappingConfig
+    mapping_config: DLiteMappingConfig
 
     def initialize(
         self, session: "Optional[Dict[str, Any]]" = None
-    ) -> "SessionUpdate":
+    ) -> DLiteSessionUpdate:
         """Initialize strategy."""
+        config = self.mapping_config.configuration
+
+        for addition in config.global_configuration_additions.storage_path:
+            dlite.storage_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.storage_plugin_path:
+            dlite.storage_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.mapping_plugin_path:
+            dlite.mapping_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.python_storage_plugin_path:
+            dlite.python_storage_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.python_mapping_plugin_path:
+            dlite.python_mapping_plugin_path.append(addition)
```

Comment:
Same as above, can be fractured out into an utility function.

Reply:
Will keep the discussion to the previous comment then, resolving this conversation.